nistats/utils.py

```diff
@@ -119,10 +120,21 @@ def _verify_events_file_uses_tab_separators(events_files):
                                     )
             except csv.Error:
                 raise ValueError(
-                    'The values in the events file are not separated by tabs; '
-                    'please enforce BIDS conventions',
-                    events_file_)
-    
+                        'The values in the events file '
+                        'are not separated by tabs; '
+                        'please enforce BIDS conventions',
+                        events_file_
+                        )
+            try:
```

Comment:
I don't understand why this block is needed now but not before.

Reply:
I was going through the docs and saw the header method and occurred to me a binary file will have readable tab separated headers, a violation of BIDS protocol, so it serves this function's purpose.